### PR TITLE
feature/way_unit_tests

### PR DESCRIPTION
**Pull request by meelros**
## JIRA
* https://ssu-jira.softserveinc.com/secure/RapidBoard.jspa?rapidView=933&view=detail&selectedIssue=PYTHON-199
* https://ssu-jira.softserveinc.com/secure/RapidBoard.jspa?rapidView=933&view=detail&selectedIssue=PYTHON-200

## Code reviewers

- [x] @BoroviyOrest
- [x] @Kardan1123
- [x] @mizin4ik
- [ ] @dimigor
- [ ] @PetrushynskyiOleksii
- [x] @Sasha1152

### Second Level Review

- [ ] @lhalam

## Summary of change
Modified way_data_validator, tests for Way model, views and add way_data_validator to way/views.py

## CHECKLIST
- [x]  СI passed
- [ ]  Сode coverage >=95%
- [x]  PR is reviewed manually again (to make sure you have 100% ready code)
- [x]  All reviewers agreed to merge the PR
- [x]  I've checked new feature as logged in and logged out a user if needed
- [x]  PR meets all conventions

### CODE REVIEW

File: way_to_home/tests/unittests/way/testmodel.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+class WayModelTestCase(TestCase):
+	"""TestCase for providing Way model testing"""
+
+	def setUp(self):
+		"""Method that provides preparation before testing Way model's features."""
+		user = CustomUser.objects.create(id=100, email='mail@gmail.com', password='Password1234', is_active=True)
+		start_place = Place.objects.create(id=100, longitude=111.123456, latitude=222.123456)
+		end_place = Place.objects.create(id=200, longitude=222.123456, latitude=111.123456)
+
```
Comment (MaxDubchak) [inline way_to_home/tests/unittests/way/testmodel.py]: longtitude must be between -180 and 180, and latitude must be betweent -90 and 90, else you might run into issues with place_data_validator

File: way_to_home/tests/unittests/way/testmodel.py [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+			user=user
+		)
+
+		self.way = Way.objects.get(id=100)
+
+		Route.objects.create(
+			id=100,
+			time='23:58:59',
+			transport_id=None,
+			position=0, way=self.way,
```
Comment (MaxDubchak) [inline way_to_home/tests/unittests/way/testmodel.py]: why are those in one line?

**Review by BoroviyOrest — DISMISSED** [review]
All is good

File: way_to_home/tests/unittests/way/testmodel.py [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+			'user_id': 100
+		}
+		actual_dict = way.to_dict()
+		self.assertDictEqual(expected_dict, actual_dict)
+
+	def test_create(self):
+		"""Provide tests for `create` method of Way model."""
+		user = CustomUser.objects.get(id=100)
+
+		way = Way.create(user=user)
```
Comment (MaxDubchak) [inline way_to_home/tests/unittests/way/testmodel.py]: In create also test passing in name

**Review by MaxDubchak — CHANGES_REQUESTED** [review]

File: way_to_home/tests/unittests/way/testmodel.py (hunk 2 above)
Comment (meelros) [reply way_to_home/tests/unittests/way/testmodel.py]: it was a mistake, changed

File: way_to_home/tests/unittests/way/testmodel.py (hunk 1 above)
Comment (meelros) [reply way_to_home/tests/unittests/way/testmodel.py]: done

File: way_to_home/tests/unittests/way/testmodel.py (hunk 3 above)
Comment (meelros) [reply way_to_home/tests/unittests/way/testmodel.py]: done

**Review by BoroviyOrest — DISMISSED** [review]

**Review by MaxDubchak — DISMISSED** [review]

**Review by Sasha1152 — DISMISSED** [review]
Good!

**Review by victoria-kit — APPROVED** [review]
Good!

**Review by lhalam — APPROVED** [review]

**Pull request merged**